scripts/prepare_automatic_backups.py

```diff
@@ -44,6 +54,11 @@ def generate_backup_url(cloud_storage_bucket_name, module_class_names):
 
 
 def update_cron_dict(cron_dict):
+    """Adding new urls in dict for backup.
+
+    Args:
+        cron_dict: dict(str, str). Content of yaml file in dictionary type.
```

Comment:
How do you know that this is a dict(str, str)? What are its keys and values?

Reply:
sorri sir, i dont know about exactly there is in yaml file,I was wrong.
but tell me what is their intit so that  can correct it.